### PR TITLE
Installed Parcel 2 plugin for static files and added Rightsidebar Kodi icon

### PR DESCRIPTION
**Pull request by developingright**
Hi @razzeee , 
I have closed the following [PR](https://github.com/xbmc/elm-chorus/pull/79) - “Added Rightsidebar Kodi icon and hover effect on slider”  because while rebasing the branch, I ran into an issue that caused a bunch of unrelated commits therefore, I thought it will be better to create a new PR and replicate the same changes in it. 

I have installed the Parcel 2 plugin for the static files and have added the Rightsidebar Kodi icon with hover-effect on the slider.

### CODE REVIEW

File: src/Components/RightSidebar.elm [hunk 1]
```diff
@@ -17,7 +18,17 @@ view { showRightSidebarMenu, showRightSidebarMenuMsg, clearPlaylistMsg, refreshP
...
             [ height (px panelHeight), width (px 400), Background.color Colors.playlistHeaderBackground, alignRight, htmlAttribute <| Html.Attributes.style "pointer-events" "all" ]
             [ row [ width fill ]
-                [ Input.button [ Background.color Colors.backgroundKodi, height (px 50), width (px 100), padding 8 ] { onPress = Just rightSidebarMsg, label = Element.text "Kodi" }
+                [ Input.button [ Background.color Colors.backgroundKodi, height (px 50), width (px 100), padding 8 ]
+                    { onPress = Just rightSidebarMsg
+                    , label =
+                        Element.row []
+                            [ image [ width (px 15), height (px 15), paddingXY 15 0 ]
+                                { description = ""
+                                , src = "http://localhost:1234/logo.png"
```
Comment (razzeee) [inline src/Components/RightSidebar.elm:27]: This url needs to be relative, when we build this for kodi (or even in dev) it's not clear, that the image will always be at that port (and host)

**Comment by developingright** [comment]
Hi @razzeee, 
 After some research, I have found the &lt;base> Html element, which specifies the base URL to use for all relative URLs contained within a document. This means that if you set &lt; base href=”http//:localhost:1234”>, any relative link I would add like 
&lt;a href=”/images/logo.png”> It will automatically point to " http://localhost:1234/images/logo.png ” and we only need to define the base URL once in the index.html file which will allow us to use relative URLs anywhere in the files.

I would love to have your suggestions on this.

**Comment by razzeee** [comment]
I think you don't even need that, you should be able to use relative paths anywhere. Base path is mostly useful when you have subpaths that are different. But feel free to play around with that.

**Comment by developingright** [comment]
Hi @razzeee,
I have made the required changes.

**Comment by razzeee** [comment]
I assume you tested and it worked?

**Comment by developingright** [comment]
@razzeee, 
Yes, I have tested it and it worked.

**Pull request merged**